docs/validation/daily-report/2023-11-27/index.md

```diff
@@ -1,8 +1,8 @@
-# 2023-11-27: Mullion, Landsat 8 overpass
 
-A Daily Validation Summary Report of the surface reflectance data collected on
-the date of 2023-11-27 by Geoscience&nbsp;Australia. The full collection of data is
-contained in the [National Spectral Database](https://www.dea.ga.gov.au/products/national-spectral-database).
+# 2023-11-27: Transect: NSW Site 1, Sentinel 2b overpass
```

Comment:
Can the double colon (`:`) be avoided here? Maybe either:

* `# 2023-11-27: Transect, NSW Site 1, Sentinel 2b overpass`
* `# 2023-11-27: Transect of NSW Site 1, Sentinel 2b overpass`